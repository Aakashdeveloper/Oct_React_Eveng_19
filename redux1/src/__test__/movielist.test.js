import React from 'react';
import ListView from '../component/moviesList';
import {create} from 'react-test-renderer';

describe('ListView Snapshot', () => {
    test("ListView",() => {
        let tree = create(<ListView/>);
        expect(tree.toJSON()).toMatchSnapshot();
    })
})
