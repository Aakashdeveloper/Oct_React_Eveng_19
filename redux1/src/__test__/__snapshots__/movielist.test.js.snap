// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListView Snapshot ListView 1`] = `<div />`;
